src/mainboard/protectli/vault_cml/vboot-rwa.fmd

```diff
@@ -0,0 +1,38 @@
+FLASH@0xff000000 0x1000000 {
```

Comment:
The best example of fmd file is here: https://github.com/Dasharo/coreboot/blob/asus_kgpe-d16/release/src/mainboard/asus/kgpe-d16/vboot-rwa-16M.fmd

You should specify regions with fixed size and provide the exact sizes. The CBFS regions should not have any sized defined to let flashmap parser spread it across whole free space. Of course you need to maintain DESC nad ME region. WP_RO must be aligned with the protected regions of SPI flash. Check the datasheet of the flash chip and verify the WP_RO length

Reply:
I have aligned the RO section with SPI flash acording to the [datasheet](https://datasheetspdf.com/pdf-file/850581/Macronix/25L12835F/1)

Reply:
Now looks good.